### ISSUE TITLE
This custom model requires an implementation named 'TCRecommender' which was not found in the global namespace

### ISSUE THREAD

**Issue by JSingh-sol** [apple/turicreate#2148]
Hi,
I want to reopen issue #1160 because the solution listed did not work, so I reason the issue has reemerged. The error is the same: Using the item similarity model, I get a "This custom model requires an implementation named 'TCRecommender' which was not found in the global namespace" when trying to use on my iPhone 7 iOS12 device. 
<img width="921" alt="Screen Shot 2019-07-19 at 9 24 57 AM" src="https://user-images.githubusercontent.com/51454748/61542426-2abc7e80-aa07-11e9-8a91-42d1a5c09cc7.png">

That is the error message. Here is the build settings showing the .dylib in correct locations : 

<img width="940" alt="Screen Shot 2019-07-19 at 9 26 17 AM" src="https://user-images.githubusercontent.com/51454748/61542516-56d7ff80-aa07-11e9-8cf1-6b5580a81795.png">

I have also tried previous versions of the dylib, both the x86 and arm64 versions.

<img width="211" alt="Screen Shot 2019-07-19 at 9 28 16 AM" src="https://user-images.githubusercontent.com/51454748/61542630-9272c980-aa07-11e9-9bf1-bb1c9ef203a3.png">

Any help would greatly be appreciated!

**Comment by JSingh-sol** [comment]
<img width="738" alt="Screen Shot 2019-07-19 at 10 49 16 AM" src="https://user-images.githubusercontent.com/51454748/61548252-e59e4980-aa12-11e9-80c9-e5a9525cd97a.png">

Here is a picture if it helps of the actual code. It breaks assigning the variable model to the mlmodel because it cannot find the dependency. So I understand this is an issue with the dylib but I have put it in all the right places!

**Comment by TobyRoseman** [comment]
@JSingh-sol - are you following the [deployment instructions in our User Guide](https://apple.github.io/turicreate/docs/userguide/recommender/coreml-deployment.html)?

**Comment by JSingh-sol** [comment]
Yes. The file has the correct name is placed in the correct locations but for some reason the error persists.

**Comment by hoytak** [comment]
Related to https://github.com/apple/turicreate/issues/2050 -- linking to track these issues together.

**Comment by JSingh-sol** [comment]
@TobyRoseman Is a solution still in the works? Do you have any questions about my screenshots?
